classes.php

```diff
@@ -0,0 +1,82 @@
+<?php
+  //Importing the vendor to use Guzzle HTTP client.
+  require_once "./vendor/autoload.php";
+
+  //Using GuzzleHttp class from the vendor folder.
+  use GuzzleHttp\Client;
+
+  define("string","https://ir-dev-d9.innoraft-sites.com");
+class FetchData {
+  /**
```

Comment:
class name starts with small case.

Reply:
@veer-91800214 this is correct.
Class name starts with uppercase.